src/components/Help/TooltipAndRTDLink/index.tsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
-import T from 'prop-types';
 
 import config from 'config';
 import Link from 'components/generic/Link';
+
 // $FlowFixMe
 import Tooltip from 'components/SimpleCommonComponents/Tooltip';
 
-import { foundationPartial } from 'styles/foundation';
 import interproTheme from 'styles/theme-interpro.css'; /* needed for custom button color*/
 import fonts from 'EBI-Icon-fonts/fonts.css';
 import ipro from 'styles/interpro-new.css';
```

Comment:
As part of the migration I'm trying to remove all imports to `'styles/theme-interpro.css'` and `'styles/interpro-new.css'`to eventually be able to get rid of them. Moving any necesary styling in those into to local files, to try to have the styles closer to the code needed. And for things that are general styling moving it to general but specific files, e.g. `'styles/summary.css'` or `'styles/blocks.css'`